jobs/JSRW_BIAS_CORRECTION_PM25

```diff
@@ -30,8 +30,15 @@ export SCRIPTSsrw="${HOMEdir}/scripts"
 #
 export USHdir="${USHsrw}"  # should be removed later
 . ${USHsrw}/source_util_funcs.sh
-for sect in user nco platform workflow nco global cpl_aqm_parm cpl_aqm_parm task_run_post \
```

Comment:
This is a much better cleanup! I've looked at this a few times now and wondered, why were these repeated originally anyway?